Docs/general update nov2023 (#67)
neicnordic/neic-sda: docs/deploy.md

The production deployment repositories are:

-   [Kubernetes Helm charts](https://github.com/neicnordic/sensitive-data-archive/tree/main/charts);
-   [Docker Swarm
    deployment](https://github.com/neicnordic/LocalEGA-deploy-swarm/).

The following container image is used in the deployments `neicnordic/sensitive-data-archive`, provides the SDA services as well as PostgreSQL and RabbitMQ configuration. The tag is what separates the services:

- `ghcr.io/neicnordic/sensitive-data-archive:<version>-postgres` - PostgreSQL database
- `ghcr.io/neicnordic/sensitive-data-archive:<version>-rabbitmq` - RabbitMQ message broker
- `ghcr.io/neicnordic/sensitive-data-archive:<version>-sftp-inbox` - sftp inbox

> - `ghcr.io/neicnordic/sensitive-data-archive:<version>-<service>` - where `<service>` corresponds to services such as: `finalize`, `ingest`, `intercept`, `verify`, `mapper`, `download`, `s3inbox`, `auth`

Reviewer: ```suggestion
- `ghcr.io/neicnordic/sensitive-data-archive:<version>-auth` - authentication service
- `ghcr.io/neicnordic/sensitive-data-archive:<version>-download` - download service
- `ghcr.io/neicnordic/sensitive-data-archive:<version>` - all other services such as: `finalize`, `ingest`, `intercept`, `verify`, `mapper` and `s3inbox`
```

The production deployment repositories are:

-   [Kubernetes Helm charts](https://github.com/neicnordic/sensitive-data-archive/tree/main/charts);
-   [Docker Swarm
    deployment](https://github.com/neicnordic/LocalEGA-deploy-swarm/).

> The following container image is used in the deployments `neicnordic/sensitive-data-archive`, provides the SDA services as well as PostgreSQL and RabbitMQ configuration. The tag is what separates the services:

Reviewer 2: ```suggestion
`neicnordic/sensitive-data-archive`, provides the SDA services as well as PostgreSQL and RabbitMQ configuration. The following container image is used in the deployments where the tag separates between services:
```
